modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -102,6 +102,55 @@ definition {
 		}
 	}
 
+	@description = "LPS-160159. Assert that it's possible to select a new custom view and selected the default view"
+	@priority = "4"
+	test CanBeSelected {
+		task ("Given FDS sample portlet deployed and given customized tab active") {
+			AssertElementPresent(
+				key_tabName = "Customized",
+				locator1 = "StagingPublishToLive#ACTIVE_TAB_NAME");
+		}
+
+		task ("When open menu to select created custom view") {
+			FDSFilters.editFilters(
+				key_disableFilters = "Blue,Green,Yellow",
+				key_enableFilters = "Red",
+				key_filter = "Color");
+
+			FDSCustomView.createNewView(key_nameView = "Custom Test");
```

Comment:
This needs to be in it's own task ("And Given a created customized view") as noted in test scenario.